widget/entry.go

```diff
@@ -820,6 +820,9 @@ func (e *Entry) rowColFromTextPos(pos int) (row int, col int) {
 				row++
 			}
 			col = pos - b[0]
+			if e.Wrapping != fyne.TextWrapOff && b[0] == pos && col == 0 && pos != 0 {
```

Comment:
This check is guarding with TextWrapOff, presumably so that the cursor moves down in the case of TextWrapBreak & TextWrapWord, but what about TextTruncate?

Reply:
I thought we did not support truncating an entry?

Reply:
```go
func (e *Entry) textWrap() fyne.TextWrap {
	if e.Wrapping == fyne.TextTruncate {
		fyne.LogError("Entry does not allow Truncation", nil)
```

Reply:
Oh yes, quite right